### PR TITLE
Mackey -- Archey-alike for OSX using system Ruby

### PR DESCRIPTION
**Pull request by docwhat**
This is yet another Archey-alike for OSX. It displays a pretty (256
color on terminals that support it) old-timey Apple logo and some
statistics about your system.

It was written by me, so feel free grouse about it. :-)

### CODE REVIEW

File: Library/Formula/mackey.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require "formula"
```
Comment (DomT4) [inline Library/Formula/mackey.rb]: Can snip this line :)

**Comment by mistydemeo** [comment]
Install failing on all platforms:

```
==> ./configure
./configure: line 57: /private/tmp/mackey-PtolQq/mackey-1.0.0/bin/bundle: No such file or directory
```

**Comment by docwhat** [comment]
@mistydemeo Heh... it looks like I didn't take into consideration that the user would have done `sudo gem install bundler` like on the Jenkins' slaves.  So doing `--conservative` meant it saw it in the system's gem list and didn't install it locally.

**Comment by docwhat** [comment]
Your Jenkins system is pretty cool... it caught some problems in my code on Mountain Lion that I wouldn't have found since I don't have any Mountain Lion boxes around (nor Mavericks for that matter).

**Comment by tdsmith** [comment]
We tend to prefer that applications explicitly declare and fetch their dependencies instead of using tools like bundler to fetch them; you can see an example of that in Pythonland in e.g. jrnl.rb.

We'll also have to decide if this meets our notability requirements; Homebrew rarely packages very new projects and self-submissions.

There's some discussion of these points in our acceptable formulæ documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md

**Comment by docwhat** [comment]
@tdsmith 

Fair enough... I was feeling that archey wasn't being updated enough and would have obscure weird bugs at times (like color not always working, etc.)  I have put some pull requests for the project but figured I could do something better in ruby.  Plus I wanted 256 colors. :-)

Re: dependencies... I'm using bundler in stand-alone mode, so the gems it downloads will only effect itself and not anything else.  This is the only way to use ruby gems reliably with the system ruby without resorting to (*shudder*) `sudo gem install`.

I can't think of an easy way to pull down gems and only install them locally. Juggling paths is hard and bundler in standalone mode makes them easy. :-/

**Comment by mistydemeo** [comment]
> I was feeling that archey wasn't being updated enough and would have obscure weird bugs at times (like color not always working, etc.)

This was actually the reason we've rejected other archey variants in the past (#29364) - there's a large number of them, and we don't really want to be in the position of picking favourites for things that don't already have community acceptance.

**Comment by docwhat** [comment]
Fair enough.

**Comment by tdsmith** [comment]
Thanks for considering Homebrew! We'll take a pass on this but you're free to host the formula yourself (you can `brew install https://some/arbitrary/url.rb`) or set up [a tap](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Interesting-Taps-%26-Branches.md) and we're happy to answer questions about either.

**Pull request closed without merging**